src/main/target/STM32F411/target.h

```diff
@@ -46,7 +46,7 @@
 #define TARGET_IO_PORTD 0xffff
 #define TARGET_IO_PORTE 0xffff
 
-#if defined(USE_RX_SPI) && !defined(CLOUD_BUILD)
+#if defined(USE_RX_SPI) || !defined(CLOUD_BUILD)
```

Comment:
This breaks compilation on
`make TARGET=STM32F411 EXTRA_FLAGS="-DCLOUD_BUILD -DUSE_SERIAL_RX -DUSE_SERIALRX_CRSF -DUSE_TELEMETRY -DUSE_TELEMETRY_CRSF -DUSE_OSD -DUSE_RX_SPI -DUSE_GYRO_SPI_ICM20689 -DUSE_ACC_SPI_ICM20689 -DUSE_ACC_SPI_MPU6000 -DUSE_GYRO_SPI_MPU6000 -DUSE_MAX7456 -DUSE_RX_EXPRESSLRS -DUSE_RX_EXPRESSLRS_TELEMETRY -DUSE_RX_SX1280 -DRX_CHANNELS_AETR -DUSE_GYRO -DUSE_ACC"
`

Reply:
Not quite. The issue was actually much deeper than that. Easier to move it to later in the file also.